scm-plugins/scm-svn-plugin/src/main/java/sonia/scm/repository/spi/SvnBrowseCommand.java

```diff
@@ -203,9 +203,18 @@ private void fetchExternalsProperty(SVNRepository repository, long revision,
     {
       SVNProperties properties = new SVNProperties();
 
-      repository.getFile(entry.getRelativePath(), revision, properties, null);
+      repository.getDir(entry.getRelativePath(), revision, properties, (Collection) null);
 
-      String externals = properties.getStringValue(SVNProperty.EXTERNALS);
+      String externals = properties.getStringValue(SVNProperty.EXTERNALS).replaceAll("[\\r\\n]+", "");
+      String[] externalsArray = externals.split(" ");
+      for(String external: externalsArray)
+      {
+        if(external.startsWith("http://") || external.startsWith("https://") || external.startsWith("../")
+          || external.startsWith("^/") || external.startsWith("/"))
```

Comment:
Please put this logic into a new function named like `shouldSetExternal()`. This would improve the readability.